Dockerfile.builder

```diff
@@ -5,32 +5,33 @@ ENV LANG=en_US.utf8
 ARG USE_GO_VERSION_FROM_WEBSITE=0
 
 # Some packages might seem weird but they are required by the RVM installer.
-RUN yum --enablerepo=centosplus install -y --quiet \
+RUN yum install epel-release --enablerepo=extras -y \
+    && yum --enablerepo=centosplus --enablerepo=epel-testing install -y \
```

Comment:
How about using `epel` instead of `epel-testing`? Also note that we don't need to use `--enablerepo=extras` when we are using `epel`

Reply:
@dipak-pawar this is same as done elsewhere [link](https://github.com/fabric8-services/fabric8-tenant/pull/696/files#diff-1562924eebb6aea9e1e174c2f95d6e4e) @MatousJobanek can you plz reply here.

Reply:
let's wait for @MatousJobanek comment as I can see he set `enablerepo=epel-testing` in all repo.

Reply:
done [b9942bf](https://github.com/fabric8-services/fabric8-env-client/pull/2/commits/b9942bf137ce61ec25926ff199bce00e04294577)